src/components/challengeWidget/challengeWidget.css

```diff
@@ -0,0 +1,32 @@
+.challengeWidget {
+  width: 700px;
+  height: 250px;
+  border: 15px solid burlywood;
+  background-color: white;
+}
+
+h1 {
+  color: black;
+  font-family: sans-serif;
+}
+p {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.first_button {
+  float: right;
+  padding: 5px;
+}
+.paragraph {
+  font-family: serif;
+}
+.cup {
```

Comment:
You are not using this class anywhere, maybe that's why it doesn't work.

Reply:
I tried to use the class but not working maybe I didn't get what do you mean, what can I do?